time_tracker_api/time_entries/time_entry_api.py

```diff
@@ -0,0 +1,132 @@
+from flask_restplus import fields, Resource, Namespace
+
+ns = Namespace('time_entries', description='API for time entries')
```

Comment:
You may want to change `time_entries` to `time-entries` as recommendation from https://restfulapi.net/resource-naming/

Reply:
True, my bad.